src/views/Lottery/components/YourPrizesCard/index.tsx

```diff
@@ -32,9 +32,16 @@ const YourPrizesCard: React.FC = () => {
   const winnings = getBalanceNumber(claimAmount)
   const isAWin = winnings > 0
 
+  const [, updateState] = useState<any>()
+  const forceUpdate = useCallback(() => updateState({}), [])
+
+  const handleAfterClaim = useCallback(() => {
+    forceUpdate()
+  }, [forceUpdate])
+
```

Comment:
I have used something similar elsewhere so I created a hook that will achieve the same result (along with some other use-cases)
https://github.com/pancakeswap/pancake-frontend/pull/876

Reply:
Waiting for that pr to be merged

Reply:
Done